ignite/templates/app/files/go.mod.plush

```diff
@@ -29,6 +30,10 @@ require (
 	google.golang.org/protobuf v1.31.0
 )
 
+require (
+	github.com/coinbase/rosetta-sdk-go/types v1.0.0 // indirect
```

Comment:
Sorry I don't understand what do you mean by 'nit, group' :sweat_smile:  but I can explain the reason of the presence of this line.

We usually don't list indirect dependencies in the go.mod template, because we always run `go mod tidy` once the code is scaffolded, and it does that very well. 

But here @Ehsan-saradar and I faced a strange issue related to go modules. If this dependency isn't formerly listed, then `go mod tidy` returns an error :

```
Error while running command go mod tidy: go: example/app imports
    [github.com/cosmos/cosmos-sdk/server](http://github.com/cosmos/cosmos-sdk/server) imports
    [cosmossdk.io/tools/rosetta](http://cosmossdk.io/tools/rosetta) imports
    [github.com/coinbase/rosetta-sdk-go/types](http://github.com/coinbase/rosetta-sdk-go/types): ambiguous import: found package [github.com/coinbase/rosetta-sdk-go/types](http://github.com/coinbase/rosetta-sdk-go/types) in multiple modules:
    [github.com/coinbase/rosetta-sdk-go](http://github.com/coinbase/rosetta-sdk-go) v0.7.9 (/root/go/pkg/mod/github.com/coinbase/rosetta-sdk-go@v0.7.9/types)
    [github.com/coinbase/rosetta-sdk-go/types](http://github.com/coinbase/rosetta-sdk-go/types) v1.0.0 (/root/go/pkg/mod/github.com/coinbase/rosetta-sdk-go/types@v1.0.0)
: exit status 1
```

I'm not sure of the reason behind this, but I suspect badly named/configured modules for rosetta.

Reply:
Yes, weird issue but we should probably do a double check on this before merging it. might be a temporary issue with dependencies.

Reply:
Good to know ty, I can keep that in mind!